### ISSUE TITLE
Value was either too large or too small for an unsigned byte.

### ISSUE THREAD

**Issue by DarKyGM** [DurtyFree/durty-cloth-tool#329]
I have the following error:
```
System.OverflowException: Value was either too large or too small for an unsigned byte.
   at System.Number.ThrowOverflowException(TypeCode type)
   at System.Convert.ToByte(String value)
   at CodeWalker.GameFiles.XmlMeta.Traverse(XmlNode node, MetaBuilder mb, MetaName type, Boolean isRoot)
   at CodeWalker.GameFiles.XmlMeta.Traverse(XmlNode node, MetaBuilder mb, MetaName type, Boolean isRoot)
   at CodeWalker.GameFiles.XmlMeta.GetMeta(XmlDocument doc)
   at ANrrkCKXTNElbZ8p640g.Yl04VCQ1NH6(XmlDocument , ANrrkCKXTNElbZ8p640g )
   at s9pUmOcxT1XsfjdvDBp.uxpTXoc6bJRx4NpBtAb.RQccbGSjK4(String  , List`1  )
   at wK3e8qcdjm0c3ifLe69.bfBIwxcmYRdjJOl2xvm.x0Cch6Ou7p(String  , String  , String  , String  )
   at wK3e8qcdjm0c3ifLe69.bfBIwxcmYRdjJOl2xvm.YGC4WPZZ1Ma(String  , String  , String  , String  )
   at YJ9XSiIhq7kJjOEPtIE.BqODX2ILVF0779tApxy.dl0dWNKp3fLssu6hkekO(Object , Object , Object , Object , Object )
   at YJ9XSiIhq7kJjOEPtIE.BqODX2ILVF0779tApxy.EFGIaJxccM(TargetResourceType  , String  , String  , String  , String  , ProgressDialog  )
```

App Version: **3.0.0.7**
Items: **2936 project items (2936C / 0D)**
Project targets: **FiveM**
DLC name: **outro_eup**
Collection name: **intro_clothes // **
Project path: **C:\Users\DarKy\Desktop\OUTRO OPTI\OutroEUP2\clothproject_outroeup2**
Output path: **C:\Users\DarKy\Desktop\OUTRO OPTI\OutroEUP2**
Disable Extra Ymt Creation: **False**
Create Subfolder For Cloth Drawable Types: **True**
Replace settings enabled: **False // DLC: NoDlc**
Save File Version: **5**
```

Cloth files:
`[Please provide cloth files (ydd, ytd), cloth project and any other related files here + describe reproduction steps]`

**Comment by DurtyFree** [comment]
Please DM me via Discord @DurtyFree and send me an export of your project, as I am not able to reproduce this issue entirely

**Comment by MarlonTgV** [comment]
Same Issue pls help

**Comment by DurtyFree** [comment]
@MarlonTgV Feel free to send me your project export via Direct Message on Discord

**Comment by ThatSake** [comment]
same problem

**Comment by DurtyFree** [comment]
Handled next update, its a game limitation hit

**Comment by DurtyFree** [comment]
Fixed in v3.1 https://github.com/DurtyFree/durty-cloth-tool/releases/tag/3.1.0.0
